modules/equativBidAdapter.js

```diff
@@ -97,6 +99,36 @@ function makeId() {
   return str;
 }
 
+/**
+ * Updates bid request with data from previous auction
+ * @param {*} req A bid request object to be updated
+ * @returns {*} Updated bid request object
+ */
+function updateFeedbackData(req) {
+  if (req?.ext?.prebid?.previousauctioninfo) {
+    req.ext.prebid.previousauctioninfo.forEach(info => {
+      if (tokens[info?.bidId]) {
+        feedbackArray.push({
+          feedback_token: tokens[info.bidId],
+          loss: info.bidderCpm == info.highestBidCpm ? 0 : 102,
+          price: info.highestBidCpm
+        });
+
+        delete tokens[info.bidId];
+      }
+    });
+
+    delete req.ext.prebid;
```

Comment:
what if other stuff is there

Reply:
Our engine currently doesn't read the `ext.prebid` data so we don't need to pass it.

Reply:
why would you want to need an update to your js and a campaign to get your pubs to upgrade should that change?

Reply:
Such a change to read that node is not scheduled in our roadmap. However, there are some other features planned so we'll have opportunities to remove that line from the code if needed. I just don't want to send useless data in the call to make it unnecessary big.